src/docs/sdk/features.mdx

```diff
@@ -268,6 +268,12 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+The `url` in breadcumbs and `$url` in span descriptions must be stripped of sensitive data like query parameters and/or username/password.
```

Comment:
Right now, query parameters are the only sensitive data we can reliably detect. Which other sensitive data can you think of removing? If any come to mind, please mention that here. This sentence leaves a lot of room for interpretation.
Please try to be as specific as possible.